### PR TITLE
ExpiringMultiPartyCreator.sol natspec

### PR DESCRIPTION
**Pull request by chrismaree**
more natspec!

### CODE REVIEW

**Comment by mrice32** [comment]
@ptare could you take a first pass on this?

File: core/contracts/financial-templates/implementation/ExpiringMultiPartyCreator.sol [hunk 1]
```diff
@@ -10,12 +9,16 @@ import "./ExpiringMultiParty.sol";
 
 /**
 @title Expiring Multi Party Contract creator.
-@notice Factory contract to create and register new instances of expiring multiparty contracts. Responsible for
-constraining the parameters used to construct a new EMP.
+@notice Factory contract to create and register new instances of expiring multiparty contracts.
```
Comment (nicholaspai) [inline core/contracts/financial-templates/implementation/ExpiringMultiPartyCreator.sol]: Should be an `*` before this right

Comment (nicholaspai) [reply core/contracts/financial-templates/implementation/ExpiringMultiPartyCreator.sol]: Same with @title

File: core/contracts/financial-templates/implementation/ExpiringMultiPartyCreator.sol [hunk 2]
```diff
@@ -58,26 +61,34 @@ contract ExpiringMultiPartyCreator is ContractCreator, Testable {
         1617235200, // 2021-04-01T00:00:00.000Z
         1619827200, // 2021-05-01T00:00:00.000Z
         1622505600, // 2021-06-01T00:00:00.000Z
-        1625097600, // 2021-07-01T00:00:00.000Z
-        1597017600 // 2020-08-10T00:00:00.000Z (Note: This is a special expiration that marks the end of the Tokyo Olympics 2020)
```
Comment (nicholaspai) [inline core/contracts/financial-templates/implementation/ExpiringMultiPartyCreator.sol]: RIP Tokyo2020

File: core/contracts/financial-templates/implementation/ExpiringMultiPartyCreator.sol (hunk 1 above)
Comment (chrismaree) [reply core/contracts/financial-templates/implementation/ExpiringMultiPartyCreator.sol]: fixed

File: core/contracts/financial-templates/implementation/ExpiringMultiPartyCreator.sol (hunk 2 above)
Comment (chrismaree) [reply core/contracts/financial-templates/implementation/ExpiringMultiPartyCreator.sol]: 💔

**Pull request merged**